kustomize/overlays/prod/shard/patches/deploy-config.yaml

```diff
@@ -4,6 +4,9 @@ data:
   EVENT_BRIDGE_K8S_PLATFORM: openshift
   EVENT_BRIDGE_MANAGER_URL: http://event-bridge:8080
   KAFKA_SECURITY_PROTOCOL: SASL_SSL
+  EVENT_BRIDGE_FM_SSO_URL: "https://keycloak-event-bridge-prod.apps.openbridge-dev.fdvn.p1.openshiftapps.com/auth/realms/event-bridge-fm/protocol/openid-connect/token"
```

Comment:
@fjtirado These configuration attributes relate to the shard, so I don't think the `_FM_` part of the name makes sense. `EVENT_BRIDGE_SSO_URL` works for me 👍

Reply:
I was trying to point out that, at the end, it is the same host name and realm than the FM one, actually, I think the manger one should be EVENT_BRIGDE_SSO_VALIDATE_URL and the Shard one EVENT_BRIDGE_SSO_OBTAIN_URL, so rather than indicating the component in which the ENV is being used, we are emphasizing its purpose, wdyt?

Reply:
@robpblake

Reply:
@fjtirado Personally, I think the name of the `ConfigMap` in which the values are stored is the most useful indicator of their purpose, so for example we will have `event-bridge-shard-operator-config` and eventually I'm guessing `event-bridge-manager-config`

I'd personally just standardise everything on `EVENT_BRIDGE_SSO_URL` and `EVENT_BRIDGE_SSO_CLIENT_ID` and `EVENT_BRIDGE_SSO_SECRET` across all configs.

Reply:
@robpblake I also like it, so removing the FM from SSO in all files